kafka-1.0.x/src/main/scala/monix/kafka/CommittableOffsetBatch.scala

```diff
@@ -87,4 +87,20 @@ object CommittableOffsetBatch {
     } else {
       empty
     }
+
+  /**
+   * Builds [[CommittableOffsetBatch]] list from offsets sequence by grouping the offsets by commitCallback.
+   *
+   * */
+  def mergeByCommitCallback(committableOffsets: Seq[CommittableOffset]): List[CommittableOffsetBatch] = {
```

Comment:
I think a test (possibly property based) should be easy to do and could be useful to ensure we don't end up with wrong offsets